.eslintignore

```diff
@@ -0,0 +1,9 @@
+# api subpackage
```

Comment:
Wonder if these should be kept in their own package.

Reply:
ESLint wasn't liking ignore files at the subpackage level 😕